server/index.js

```diff
@@ -23,14 +24,19 @@ app.use(jsonMiddleware);
 
 app.use(errorMiddleware);
 
+app.use(authorizationMiddleware);
 app.get('/api/users', (req, res) => {
-
+  let userId = 1;
+  if (req.user) {
+    userId = req.user.userId;
+  }
```

Comment:
You shouldn't need these checks in your endpoints, the middleware will prevent the code from even getting here is a valid token wasn't supplied. The point of the auth middleware is to prevent people from accessing data if they aren't signed in. So here you would just need:

```JAVASCRIPT
const userId = req.user.userId;
```

This applies for all the endpoints with userIds

Reply:
![users-can-sign-in](https://user-images.githubusercontent.com/71291742/144501696-c9a2338e-e19a-496d-bc4f-cbad8726d1cd.gif)